core/00_primordials.js

```diff
@@ -530,6 +580,17 @@
   // This is because the identifier `eval` is somewhat treated as a keyword
   primordials.indirectEval = eval;
 
+  // Removing unsafe functions
+  [
+    "PromiseAll",
+    "PromiseAllSettled",
+    "PromiseAny",
+    "PromiseRace",
+    "PromisePrototypeFinally",
+  ].forEach((name) => {
+    delete primordials[name];
+  });
```

Comment:
We can overwrite unsafe functions like this:

```
primordials.PromiseAll = SafePromiseAll;
```

@littledivy How do you think?